site/guide/_manage-permissions.qmd

```diff
@@ -0,0 +1,3 @@
+1. Under User Permissions, select **Permissions**
+
+1. Under your desired action, add or delete roles. 
```

Comment:
```suggestion
1. Add or delete roles that hold a specific permission:
```

I would then include the bulleted list you have in your comment/question and add an example below that: 

```
   - To assign a role to a permission, blah blah blah
   - To remove a role from a permission, blah blah blah

   For example, to grant a developer the permission to update an existing template ... yada yada yada 
```

Also, wording like `Under your desired action` doesn't really add anything, I'd just skip in favour of direct wording — what do I desire if I am a new user?